### ISSUE TITLE
samsung printer driver fails with "Unable to execute pdftocairo for form flattening!"

### ISSUE THREAD

**Issue by peti** [NixOS/nixpkgs#49380]
I tried printing a page on my ml-2165w Samsung printer with cups running a current version of nixos-unstable, but cups fails saying:

    Unable to execute pdftocairo for form flattening!

**Comment by emmanuelrosa** [comment]
I used to have a Samsung ML-2010. Are you using the splix driver?

**Comment by peti** [comment]
No, I'm using the unified samsung driver. (Or rather, *was*, since I switched to Splix in the meanwhile.)

**Comment by Mindavi** [comment]
I feel like it isn't very useful to keep this open, is it?
